docs/Change_Log.md

```diff
@@ -1,5 +1,46 @@
 # Change Log
 
+## API and Gateway V8
+
+#### September 24, 2020
+
+We've introduced API and Gateway v8! Changes are noted throughout the documentation, but the major changes are:
+
+- API v8 is now available. v6 is still the default for the time being.
+- All permissions have been converted to strings-serialized numbers. As such, `permissions_new`, `allow_new`, and `deny_new` have been removed
+- The `game` field has been removed. If you need a direct replacement, you can instead reference the first element of `activities`
+- Channel Permission Overwrite `type`s are now numbers (0 and 1) instead of strings ("role" and "member"). However due to a current technical constraint, they are string-serialized numbers in audit logs.
+- `embed_enabled` and `embed_channel_id` have been removed. Use `widget_enabled` and `widget_channel_id` instead.
+- Form body errors have been improved to include more helpful messaging on validation. [See more here](#DOCS_REFERENCE/error-messages)
+- The `Retry-After` header is now based in seconds instead of milliseconds (e.g. `123` means 123 seconds)
+- The `X-RateLimit-Precision` header is no longer respected. `X-RateLimit-Reset` and `X-RateLimit-Reset-After` are always returned at millisecond precision (e.g. `123.456` instead of `124`)
+- Bots no longer receive [Channel Create Gateway Event](#DOCS_GATEWAY/channel-create) for DMs
```

Comment:
I'm curious, what's the reasoning behind this change? Was it just getting that expensive to send these to bots constantly?

Reply:
iirc gateway v6 sent a channel_create every time a dm was sent, even if in an existing dm

Reply:
oooo right, I completely forgot about that >w<